apps/builder/app/builder/features/style-panel/sections/backgrounds/background-layers.ts

```diff
@@ -201,11 +246,24 @@ export const addLayer = (
   style: StyleInfo,
   createBatchUpdate: CreateBatchUpdate
 ) => {
+  const batch = createBatchUpdate();
   const layerCount = getLayerCount(style);
-  setLayerProperty(layerCount, style, createBatchUpdate)("backgroundImage")({
-    type: "keyword",
-    value: "none",
-  });
+
+  const layerStyles = normalizeLayers(style, layerCount, batch);
+
+  for (const property of layeredBackgroundProps) {
+    const newPropertyStyle = layerStyles[property];
+
+    const newValue = [...newPropertyStyle.value];
+
+    newValue.splice(0, 0, layeredBackgroundPropsDefaults[property]);
```

Comment:
isn't this just this?
```suggestion
    const newValue = [...layeredBackgroundPropsDefaults[property]), ...newPropertyStyle.value];
```

Reply:
Yes but everythere here in code splice is used for inserts. For consistency I use splice here too

Reply:
I think its a bit confusing to use splice when you can express it simpler

Reply:
But in other places in same file splice is used as simpler option. Here basically same operation as in all other places.